### ISSUE TITLE
Sentient Potion Notification won't show for everyone (DAMN YOU MOTHBLOCKS)

### ISSUE THREAD

**Issue by SethLafuente** [tgstation/tgstation#63486]
## Reproduction: Be a ghost with all antags on and notifications and wait to not get notified after the potion is used both for xenobiology and nuke ops 

A large portion of ghost and observers including me have encountered this issue which isn't very clear unless you are actively watching xeno that you wont get notified if a sentient potion is used on a creature

I assumed this has lasted for so long since you can't report something you literary can't see but I have noticed through this months talked to admins about it no one has fixed it so reporting it here 

I don't know shit about coding but I suspect this has to do with the new character setup menu before you could decide to have sentient potions on an off now you can't so it can be that is just turned off to everyone who had it off before and theres no way to turn it back on

It probably affected me that one time admins decided to re roll everyones characters reverting all the changes they had pretty much putting all my preferences as antag to no which I can turn back on but I can't really turn sentient creatures back on since it isnt an option.

### TLDR

**Make sentient potion spawns something you can turn off an on in the antag menu**

big oversight not sure why it wasn't like this already

**Comment by Ryusenshu** [comment]
For a while i had every antag disabled via the menu, but later when i started playing antag again i never got a roll for over a week, even when dead. Sentient potion spawn included

Also just a few hours ago, someone in deadchat said they didnt have heretic enabled for like in forever, but they got it anyway

I think this problem "might" be somewhat related to that

**Comment by Mothblocks** [comment]
Duplicate of #61561

That being said, I was looking into this last night, but am prioritizing an optimization that means that adding new icons for things like this doesn't necessitate extra entire seconds onto initialization time, so it'll be a bit.
